scripts/covid_hosp_explore.R

```diff
@@ -137,525 +137,594 @@ forecaster_parameter_combinations_ <- rlang::list2(
-        )
-        state_id <- readr::read_csv(
-          "https://raw.githubusercontent.com/cmu-delphi/covidcast-indicators/refs/heads/main/_delphi_utils_python/delphi_utils/data/2020/state_codes_table.csv"
-        )
-        hhs_region %>%
-          left_join(state_id, by = "state_code") %>%
-          select(hhs_region = hhs, geo_value = state_id) %>%
-          mutate(hhs_region = as.character(hhs_region))
+      # Sum the two signals.
+      google_symptoms_archive$DT %>%
+        mutate(
+          google_symptoms = ifelse(is.na(google_symptoms_4_bronchitis), 0, google_symptoms_4_bronchitis) +
+            ifelse(is.na(google_symptoms_5_ageusia), 0, google_symptoms_5_ageusia)
+        ) %>%
+        select(-starts_with("source")) %>%
+        # Always convert to data.frame after dplyr operations on data.table
+        # https://github.com/cmu-delphi/epiprocess/issues/618
+        as.data.frame() %>%
+        as_epi_archive(compactify = TRUE)
       }
     ),
-    tar_target(
-      name = joined_archive_data,
-      command = {
-        # reformt hhs_archive, remove data spotty locations
-        joined_archive_data <- hhs_archive$DT %>%
-          select(geo_value, time_value, value, version) %>%
-          rename("hhs" := value) %>%
-          add_hhs_region_sum(hhs_region) %>%
-          filter(geo_value != "us") %>%
-          # Always convert to data.frame after dplyr operations on data.table
-          # https://github.com/cmu-delphi/epiprocess/issues/618
-          as.data.frame() %>%
-          as_epi_archive(compactify = TRUE)
-        joined_archive_data$geo_type <- "custom"
-        # drop aggregated geo_values
-        joined_archive_data <- joined_archive_data %>% epix_merge(nwss_coarse, sync = "locf")
-        joined_archive_data %<>% epix_merge(nssp_archive, sync = "locf")
-        joined_archive_data$geo_type <- "custom"
-        joined_archive_data %<>% epix_merge(google_symptoms_archive, sync = "locf")
-        joined_archive_data <- joined_archive_data$DT %>%
-          filter(grepl("[a-z]{2}", geo_value), !(geo_value %in% c("as", "pr", "vi", "gu", "mp"))) %>%
-          # Always convert to data.frame after dplyr operations on data.table
-          # https://github.com/cmu-delphi/epiprocess/issues/618
-          as.data.frame() %>%
-          as_epi_archive(compactify = TRUE)
-        joined_archive_data$geo_type <- "state"
-        slide_forecaster(
-          epi_archive = joined_archive_data,
-          outcome = "hhs",
-          ahead = 0,
-          forecaster = dummy_forecaster,
-          n_training_pad = 30L,
-          forecaster_args = list(),
-          forecaster_args_names = list(),
-          ref_time_values = ref_time_values,
-          start_date = start_date,
-          end_date = end_date,
-          date_range_step_size = date_step,
-          cache_key = "joined_archive_data"
-        )
-        joined_archive_data
-      }
-    )
+  tar_target(
+    name = state_to_hhs_crosswalk,
```

Comment:
this appears to be unused?

Reply:
Yea it's some geo stuff I started refactoring, but decided to move to another branch. Let me see if I can pull this out too.

Reply:
moved